Frontend/src/App.css

```diff
@@ -0,0 +1,53 @@
+.App {
```

Comment:
This file is from create-react-app right? If you are not using these styles they could also be removed